css/src/atomics/border.scss

```diff
@@ -0,0 +1,55 @@
+.border {
```

Comment:
Can confirm output here looks like what I'd expect:


```css
.border {
	border: 1px solid var(--theme-border) !important;
}
.border-none {
	border: 0 !important;
}
@media print, screen and (min-width: 768px) {
	.border-tablet {
		border: 1px solid var(--theme-border) !important;
	}
	.border-none-tablet {
		border: 0 !important;
	}
}
.border-top {
	border-top: 1px solid var(--theme-border) !important;
}
.border-top-none {
	border-top: 0 !important;
}
@media print, screen and (min-width: 768px) {
	.border-top-tablet {
		border-top: 1px solid var(--theme-border) !important;
	}
	.border-top-none-tablet {
		border-top: 0 !important;
	}
}
.border-right {
	border-right: 1px solid var(--theme-border) !important;
}
.border-right-none {
	border-right: 0 !important;
}
@media print, screen and (min-width: 768px) {
	.border-right-tablet {
		border-right: 1px solid var(--theme-border) !important;
	}
	.border-right-none-tablet {
		border-right: 0 !important;
	}
}
.border-bottom {
	border-bottom: 1px solid var(--theme-border) !important;
}
.border-bottom-none {
	border-bottom: 0 !important;
}
@media print, screen and (min-width: 768px) {
	.border-bottom-tablet {
		border-bottom: 1px solid var(--theme-border) !important;
	}
	.border-bottom-none-tablet {
		border-bottom: 0 !important;
	}
}
.border-left {
	border-left: 1px solid var(--theme-border) !important;
}
.border-left-none {
	border-left: 0 !important;
}
@media print, screen and (min-width: 768px) {
	.border-left-tablet {
		border-left: 1px solid var(--theme-border) !important;
	}
	.border-left-none-tablet {
		border-left: 0 !important;
	}
}
.border-high-contrast {
	border: 1px solid var(--theme-border-white-high-contrast);
}
.border-high-contrast-hover:hover {
	border: 1px solid var(--theme-border-yellow-high-contrast);
}
.border-radius {
	border-radius: 0.25rem !important;
}
.border-radius-l {
	border-radius: 0.5rem !important;
}

```

Reply:
I did notice that, compared to spacing, media queries not collected together. I think that, although very small, this may increase the output a bit. Is there a way to author these so that media queries are collected together like in spacing?


```css

@media print, screen and (min-width: 768px) {
	.border-bottom-tablet {
		border-bottom: 1px solid var(--theme-border) !important;
	}
	.border-bottom-none-tablet {
		border-bottom: 0 !important;
	}
}
// 👆 can this
.border-left {
	border-left: 1px solid var(--theme-border) !important;
}
.border-left-none {
	border-left: 0 !important;
}
// 👇be placed within the same media query as this?
@media print, screen and (min-width: 768px) {
	.border-left-tablet {
		border-left: 1px solid var(--theme-border) !important;
	}
	.border-left-none-tablet {
		border-left: 0 !important;
	}
}
```